util/netevent.c

```diff
@@ -833,6 +846,8 @@ comm_point_udp_ancil_callback(int fd, short event, void* arg)
 #ifndef S_SPLINT_S
 	struct cmsghdr* cmsg;
 #endif /* S_SPLINT_S */
+	struct timespec *ts;
+	struct timeval *tv;
```

Comment:
The compiler says this is an unused variable.
```suggestion
```

Reply:
Ah, yeah, it's an artefact of the previous version, we can safely remove it.